config/resources/dossier-domain.lisp

```diff
@@ -5,7 +5,9 @@
                 (:created       :datetime ,(s-prefix "dct:created"))
                 (:number        :number   ,(s-prefix "adms:identifier")))
   :has-one `((decisionmaking-flow  :via      ,(s-prefix "dossier:Dossier.isNeerslagVan")
-                    :as "decisionmaking-flow"))
+                    :as "decisionmaking-flow")
+             (parliament-flow  :via      ,(s-prefix "parl:behandeltDossier")
+                    :as "parliament-flow"))
```

Comment:
While reviewing this I wondered "wouldn't this make more sense as an inverse, like in `sign-flow`"? Then checked the story and realized I put it in there like this myself 🤦‍♂️ . Sorry about that! Will adapt this and the other PRs to switch this to inverse.